6_visualize.yml

```diff
@@ -48,13 +56,40 @@ targets:
       tmp=I('6_visualize/tmp'),
       log=I('6_visualize/log'))
 
+  6_vizprep/out/view_fun.rds.ind:
```

Comment:
I'd prefer it if the .yml filename and the directory of these outputs matched. Renaming the .yml to 6_vizprep.yml would be one option, and I think the still-better option would be, given that we're generating both preparatory functions and final png files within this .yml, that we put all the outputs into 6_visualize.

You and I probably agree that it's too many files to be dumping into a single `out` folder all together, so I propose that we create subfolders within `6_visualize/out`. To get super organized, we could have:
* `6_visualize/out/prep_static` (for view_fun.rds, basemap_fun.rds, storm_line_fun.rds, rivers_fun.rds, and watermark_fun.rds, and their corresponding .ind files)
* `6_visualize/out/prep_dynamic` (for all xx_fun.rds that are timestep specific)
* `6_visualize/out/frames` (for all .png output files)
* `6_visualize/out/gif` (for the final thing)

Or we could go with something a little simpler if you prefer - I don't care much about how we divide the subfolders, really just want that top folder to be aligned with the yml. A simpler example:
* `6_visualize/out/prep`
* `6_visualize/out/final`

Reply:
I agree, and I was planning to move things around to avoid that folder. But I also considered my approach to be pretty flawed (hence this PR and questions in it), so I didn't want to re-organize w/o input on the overall approach.

Reply:
OK great. I was hoping this wasn't going to be an extended discussion =). But know that I'd be willing to have it if you didn't agree.